docs/cookbook/getParseableZonedStringAtInstant.mjs

```diff
@@ -1,20 +1,42 @@
 import assert from 'assert';
 
 /**
- * Converts an absolute point in time to a readable string of a specified time zone
+ * Converts an absolute point in time to a string with the time in the specified
+ * time zone and a time zone designation.
  *
  * @param {Temporal.Absolute} absolute An absolute point in time
- * @param {Temporal.TimeZone|string} absolute An absolute point in time
- * @returns {string} IANA Timezone
+ * @param {string} ianaTimeZoneName IANA name of a time zone
+ * @returns {string} String indicating the time with time zone designation
  */
-function getParseableZonedStringAtInstant(absolute, ianaTimeZoneName) {
-  // inTimeZone accepts either a timezone string or a Temporal.TimeZone object
-  // It returns a DateTime object
-  const localDateTime = absolute.inTimeZone(ianaTimeZoneName);
-  return localDateTime.toString();
+function getParseableIanaZonedStringAtInstant(absolute, ianaTimeZoneName) {
+  const timeZoneObject = Temporal.TimeZone.from(ianaTimeZoneName);
+  return getParseableZonedStringAtInstant(absolute, timeZoneObject);
+}
+
+/**
+ * Converts an absolute point in time to a string with the time in the specified
+ * time zone and a time zone designation.
+ *
+ * @param {Temporal.Absolute} absolute An absolute point in time
+ * @param {string} absolute An absolute point in time
```

Comment:
This parameter is misdocumented.
```suggestion
 * @param {Temporal.TimeZone} timeZone The target time zone
```

Reply:
Thanks, opened #489 as a follow up for this.